### DISCUSSION TITLE
I found a link error in the documentation.

### DISCUSSION THREAD

**Discussion by otowa1** [wpengine/faustjs#1582] [General]
Hi. Faust.js is great. I would like to use it on a production site from now on.

https://faustjs.org/guide/creating-a-custom-block

The "Faust Scaffold" link on the page above goes to localhost:3000.  [Sprint 24 Update](https://faustjs.org/sprint-24-update#faustjs-scaffold) seems to be correct.
Sorry for the details.

**Comment by mindctrl** [comment]
@otowa552 thank you for reporting this! I've updated the link in WordPress and it will be live very shortly.
